README.md

```diff
@@ -5,13 +5,12 @@ Node.js API**  also known as **N-API**, providing C++ object model and exception
 handling semantics with low overhead. It guarantees backward compatibility with
 use with older versions of Node.js that do not have N-API built-in.
 
```

Comment:
It guarantees backward compatibility when  used with older versions of Node.js that do not have N-API built-in.